bin/semver

```diff
@@ -0,0 +1,448 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: Apache-2.0
+#
+# From https://raw.githubusercontent.com/fsaintjacques/semver-tool/3c76a6f9d113f4045f693845131185611a62162e/src/semver
```

Comment:
It might be needed to add the license section of Apache 2.0 license (as shown here: https://github.com/fsaintjacques/semver-tool/blob/master/LICENSE#L189) into this file and adjust the year and name of copyright owner.

Reply:
Or change the URL to https://github.com/fsaintjacques/semver-tool/blob/3c76a6f9d113f4045f693845131185611a62162e/src/semver

Reply:
Done